tex/gregoriotex.lua

```diff
@@ -267,9 +267,23 @@ local function get_gregorioversion()
   return internalversion
 end
 
+local function map_fonts()
+  -- use greciliae because it is the most complete font
+  local font_info = fontloader.open(kpse.find_file('greciliae', 'truetype fonts'))
```

Comment:
Finding this discussion is difficult, so I'm going to continue this discussion in #279.